src/Html.php

```diff
@@ -490,15 +490,15 @@ public static function resetButton(string $content = 'Reset', array $options = [
      *
      * @param string $type the type attribute.
      * @param string $name the name attribute. If it is null, the name attribute will not be generated.
-     * @param string $value the value attribute. If it is null, the value attribute will not be generated.
+     * @param mixed $value the value attribute. If it is null, the value attribute will not be generated.
```

Comment:
I think `mixed` isn't correct type. Type can't be `resource` or `array` 
Type can be `string`, `bool`, `int`, `float`, `null`, `stringable object` and `callable` for lazy value.

Reply:
done.